File: CalculatorAndroidStudio/app/src/main/java/com/practices/samuel/calculatorandroidstudio/MainActivity.java
package com.practices.samuel.calculatorandroidstudio;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

import org.w3c.dom.Text;

public class MainActivity extends AppCompatActivity {

    private EditText Operand1Txt;
    private EditText Operand2Txt;
    private Button AdditionButton;
    private Button SubtractionButton;
    private Button DivisionButton;
    private Button MultiplicationButton;
    private Button ClearButton;
    private TextView ResultTxt;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Operand1Txt = (EditText) findViewById(R.id.editOperand1);
        Operand2Txt = (EditText)findViewById(R.id.editOperand2);
        AdditionButton = (Button) findViewById(R.id.Addition);
        SubtractionButton = (Button) findViewById(R.id.Subtraction);
        DivisionButton = (Button) findViewById(R.id.Division);
        MultiplicationButton = (Button) findViewById(R.id.Multiplication);
        ClearButton = (Button) findViewById(R.id.Clear);
        ResultTxt = (TextView) findViewById(R.id.Result);
        CalculatorElements elements = new CalculatorElements(Operand1Txt,Operand2Txt,AdditionButton,SubtractionButton,DivisionButton,MultiplicationButton,ClearButton,ResultTxt);
        SimpleCalculator.Make(MainActivity.this,elements);

    }
}

File: CalculatorXamarin/CalculatorXamarin/MainActivity.cs
using System;
using Android.App;
using Android.Content;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.OS;

namespace CalculatorXamarin
{
    [Activity(Label = "CalculatorXamarin", MainLauncher = true, Icon = "@drawable/icon")]
    public class MainActivity : Activity
    {
        private EditText Operand1Txt;
        private EditText Operand2Txt;
        private Button AdditionButton;
        private Button SubtractionButton;
        private Button DivisionButton;
        private Button MultiplicationButton;
        private Button ClearButton;
        private TextView ResultTxt;
        protected override void OnCreate(Bundle bundle)
        {
            base.OnCreate(bundle);

           
            // Set our view from the "main" layout resource
            SetContentView(Resource.Layout.Main);
            // FindViewById(Re)
            Operand1Txt = FindViewById<EditText>(Resource.Id.editOperand1);
            Operand2Txt = FindViewById<EditText>(Resource.Id.editOperand2);
            AdditionButton = (Button)FindViewById(Resource.Id.Addition);
            SubtractionButton = (Button)FindViewById(Resource.Id.Subtraction);
            DivisionButton = (Button)FindViewById(Resource.Id.Division);
            MultiplicationButton = (Button)FindViewById(Resource.Id.Multiplication);
            ClearButton = (Button)FindViewById(Resource.Id.Clear);
            ResultTxt = (TextView)FindViewById(Resource.Id.Result);

            CalculatorElements elements = new CalculatorElements(
                Operand1Txt, Operand2Txt, AdditionButton, SubtractionButton, DivisionButton, MultiplicationButton, ClearButton, ResultTxt);
            SimpleCalculator.Make(this, elements);
        }
    }
}

